TidepoolOnboardingKitUI/Support/OnboardingSection.swift

```diff
@@ -0,0 +1,115 @@
+//
+//  OnboardingSection.swift
+//  TidepoolOnboardingKitUI
+//
+//  Created by Darin Krauss on 3/4/21.
+//  Copyright © 2021 Tidepool Project. All rights reserved.
+//
+
+enum OnboardingSection: String, CaseIterable, Codable, Equatable {
+    case welcome
```

Comment:
Should there be a comment that this order is important and needs to be maintained? Someone like me might try to order alphabetically.